### PR TITLE
Admin app. Records List directive

### PR DESCRIPTION
**Pull request by f3arnil**
1. "Beta version" of directive
2. Config as app.constant
3. New module "style"

### CODE REVIEW

File: angular-spa-ui/js/admin-app/admin-app.js [hunk 1]
```diff
@@ -6,12 +6,14 @@ var uiRouter = require('angular-ui-router');
 var uiBs = require('angular-ui-bootstrap');
 
 var adminApp = angular.module('app', [
-    uiRouter, 
-    uiBs
+    uiRouter,
+    uiBs,
```
Comment (tarhunakau) [inline angular-spa-ui/js/admin-app/admin-app.js]: what is uiBs? names MUST be obvious

File: angular-spa-ui/js/admin-app/content-manager-module/contentManager-srv.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+module.exports = function () {
+
+    var generateQueryParams = function (path, stateParams) {
+        var url = path;
+        for (var x in stateParams) {
```
Comment (tarhunakau) [inline angular-spa-ui/js/admin-app/content-manager-module/contentManager-srv.js:7]: Next time use standard array functions like join() and map()

File: angular-spa-ui/js/admin-app/content-manager-module/recordsListDir/recordsList-config.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            { title : '20 Results/page', value : '20' },
+            { title : '25 Results/page', value :'25' },
+            { title : '30 Results/page', value : '30' }
+        ],
+        itemConfig : {
+            addNew : false,
+            delete : false,
+            edit : false
+        }
+    }
```
Comment (tarhunakau) [inline angular-spa-ui/js/admin-app/content-manager-module/recordsListDir/recordsList-config.js]: As I see you did not review your pull request on your own

File: angular-spa-ui/js/admin-app/content-manager-module/recordsListDir/recordsList-srv.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
+"use strict";
+
+module.exports = function (configService) {
+
+    var findValueId = function (val, array) {
+        for (var x in array) {
```
Comment (tarhunakau) [inline angular-spa-ui/js/admin-app/content-manager-module/recordsListDir/recordsList-srv.js:6]: Next time use Array.map()

File: angular-spa-ui/js/admin-app/index.jade [hunk 5]
```diff
@@ -19,6 +19,7 @@ html
 
         include ../common/header/header.jade
         include ../common/footer/footer.jade
+        include content-manager-module/index.jade
```
Comment (tarhunakau) [inline angular-spa-ui/js/admin-app/index.jade]: Try it with crossplatform path: ./content-manager-module/index.jade

File: angular-spa-ui/js/common/app-config.js [hunk 6]
```diff
@@ -1,21 +1,24 @@
...
+        publicationDetail: '/service/publication/',
+        articleDetail: '/service/article/'
+    },
+    methods: {
+        GET: 'GET',
+        POST: 'POST',
+        PUT: 'PUT',
+        DELETE: 'DELETE'
+    },
+    userData: {
```
Comment (tarhunakau) [inline angular-spa-ui/js/common/app-config.js:18]: Do you still need it?

File: angular-spa-ui/js/search-app/search-module/search-ctrl.js [hunk 7]
```diff
@@ -141,11 +141,12 @@ module.exports = function (search) {
         $scope.sortParams = config.sortParams;
         $scope.resultsPerPages = config.resultsPerPage;
         $scope.searchInList = config.searchIn;
-        
+        console.log(searchStorage.params);
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-module/search-ctrl.js]: remove

File: angular-spa-ui/js/common/app-config.js (hunk 6 above)
Comment (f3arnil) [inline angular-spa-ui/js/common/app-config.js:18]: still need. common services is my next goal for next pull request

**Comment by tarhunakau** [comment]
Pending for review by Valery

**Comment by tarhunakau** [comment]
Could be merged

**Pull request merged**